xpdan/glbl.py

```diff
@@ -108,46 +133,44 @@ def make_glbl(env_code=0, db=None):
     ]
 
     # only create dirs if running test
-    if int(env_code) ==1:
+    if int(env_code) == 1:
         for folder in ALL_FOLDERS:
             os.makedirs(folder, exist_ok=True)
 
     # directories that won't be tar in the end of beamtime
     _EXCLUDE_DIR = [HOME_DIR, BLCONFIG_DIR, YAML_DIR]
-    _EXPORT_TAR_DIR = [CONFIG_BASE, USERSCRIPT_DIR]
-
-    class Glbl:
-        beamline_host_name = BEAMLINE_HOST_NAME
-        base = BASE_DIR
-        home = HOME_DIR
-        _export_tar_dir = _EXPORT_TAR_DIR
-        xpdconfig = BLCONFIG_DIR
-        import_dir = IMPORT_DIR
-        config_base = CONFIG_BASE
-        tiff_base = TIFF_BASE
-        usrScript_dir = USERSCRIPT_DIR
-        usrAnalysis_dir = ANALYSIS_DIR
-        yaml_dir = YAML_DIR
-        bt_dir = BT_DIR
-        sample_dir = SAMPLE_DIR
-        experiment_dir = EXPERIMENT_DIR
-        scanplan_dir = SCANPLAN_DIR
-        allfolders = ALL_FOLDERS
-        archive_dir = USER_BACKUP_DIR
-        owner = OWNER
-        beamline_id = BEAMLINE_ID
-        group = GROUP
-        det_image_field = DET_IMAGE_FIELD
-        dark_field_key = DARK_FIELD_KEY
-        calib_config_name = CALIB_CONFIG_NAME
-        exp_db = db
-        # default masking dict
-        mask_dict = {'edge': 30, 'lower_thresh': 0.0,
-                     'upper_thresh': None, 'bs_width': 13,
-                     'tri_offset': 13, 'v_asym': 0,
-                     'alpha': 2.5, 'tmsk': None}
-
-    return Glbl
+    glbl = dict(beamline_host_name=BEAMLINE_HOST_NAME,
+                base=BASE_DIR,
+                home=HOME_DIR,
+                _export_tar_dir=[CONFIG_BASE, USERSCRIPT_DIR],
+                xpdconfig=BLCONFIG_DIR,
+                import_dir=IMPORT_DIR,
+                config_base=CONFIG_BASE,
+                tiff_base=TIFF_BASE,
+                usrScript_dir=USERSCRIPT_DIR,
+                usrAnalysis_dir=ANALYSIS_DIR,
+                yaml_dir=YAML_DIR,
+                bt_dir=BT_DIR,
+                sample_dir=SAMPLE_DIR,
+                experiment_dir=EXPERIMENT_DIR,
+                scanplan_dir=SCANPLAN_DIR,
+                allfolders=ALL_FOLDERS,
+                archive_dir=USER_BACKUP_DIR,
+                owner=OWNER,
+                beamline_id=BEAMLINE_ID,
+                group=GROUP,
+                det_image_field=DET_IMAGE_FIELD,
+                dark_field_key=DARK_FIELD_KEY,
+                calib_config_name=CALIB_CONFIG_NAME,
+                exp_db=db1,
+                an_db=db2,
+                mask_dict={'edge': 30, 'lower_thresh': 0.0,
```

Comment:
here we are hard-coding default numbers in glbl.py....is that a good idea?  wouldn't it be better for all numbers to be in the config?  Some of these numbers could change if we get a new detector, for example.